### ISSUE TITLE
NA Birds 2

### ISSUE THREAD

**Issue by samuelstevens** [samuelstevens/swin-transformer#2]
Dataset: NA Birds
Data: 10%
Model: Best iNat21 (no hierarchy)

Use a pre-trained swin-v2-base model pre-trained on iNat21 with the best validation loss, fine-tune it on 10% of the NA Birds training data, then report the accuracy (and preferably attach training logs, link to WandB dashboards, etc).
